sensor/kubernetes/complianceoperator/updater.go

```diff
@@ -29,9 +32,40 @@ const (
 )
 
 var (
-	log = logging.LoggerForModule()
+	log  = logging.LoggerForModule()
+	once = sync.Once{}
 )
 
+// registerDiagnosticComplianceOperatorObjects adds compliance operator objects to the default objects to pull in the diagnostic bundles
+func (u *updaterImpl) registerDiagnosticComplianceOperatorObjects(info *central.ComplianceOperatorInfo) {
+	log.Infof("Add compliance operator objects to diagnostic bundles with namespace: %s", info.GetNamespace())
+	registerFunc := func(cfg k8sintrospect.Config) k8sintrospect.Config {
+		cfg.Objects = append(cfg.Objects, k8sintrospect.ObjectConfig{
```

Comment:
Missing rules.

Reply:
I've intentionally not added them to save some MBs in the diagnostic bundle and they are static. Since they are added always  by CO I've not seen to much value in adding them.

Reply:
Then what about profiles?  Those are static.  We should probably treat profiles and rules the same.

Reply:
I've added profiles because they are few. I am not sure how big a diagnostics bundle can get, adding rules would add 3MB size per cluster to the bundle. I am asking in Slack what others think.

Reply:
well I think that is a valid reason to exclude rules then.